Classes/Controller/AbstractTopicController.php

```diff
@@ -12,11 +12,11 @@
 namespace JWeiland\Pforum\Controller;
 
 use JWeiland\Pforum\Domain\Model\Forum;
-use JWeiland\Pforum\Domain\Model\FrontendUser;
 use JWeiland\Pforum\Domain\Model\Topic;
 use TYPO3\CMS\Core\Mail\MailMessage;
 use TYPO3\CMS\Core\Messaging\FlashMessage;
 use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+use TYPO3\CMS\Extbase\Domain\Model\FrontendUser;
```

Comment:
Durch diese Änderung hast Du soeben alle Validatoren für die FE Eingabe rausgeschmissen ;-)

Reply:
Da war ich bei den ganzen Extensions wohl zu schnell und hab das extends User in der FrontendUser Klasse überlesen. Danke ^^ reverted